extensions/ollama/src/extension.ts

```diff
@@ -0,0 +1,39 @@
+/**********************************************************************
+ * Copyright (C) 2025 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import type { ExtensionContext } from '@kortex-app/api';
+
+import { OllamaExtension } from './ollama-extension';
+
+let ollamaExtension: OllamaExtension | undefined;
+
+export async function activate(extensionContext: ExtensionContext): Promise<void> {
+  ollamaExtension ??= new OllamaExtension(extensionContext);
+  await ollamaExtension.activate();
+}
+
+export async function deactivate(): Promise<void> {
+  await ollamaExtension?.deactivate();
+  ollamaExtension = undefined;
+}
+
+// Expose ollamaExtension for testing purposes
+if (process.env.NODE_ENV === 'test') {
+  Object.defineProperty(global, 'ollamaExtension', {
+    get: () => ollamaExtension,
+  });
+}
```

Comment:
Why are you defining the global property `ollamaExtension` in test mode, this is not something I would expect in an extension.

Reply:
I tried to remove this code, and the tests are still passing so I think it can be safely removed.

<img width="787" height="194" alt="Image" src="https://github.com/user-attachments/assets/5e50330e-bee4-4c38-a5ee-d88d816ccfd3" />

<img width="787" height="194" alt="Image" src="https://github.com/user-attachments/assets/48686b28-5f34-4fc5-b75d-e3521716282d" />

Reply:
ps: there are more than 3 tests